docs/zh/latest/discovery/eureka.md

```diff
@@ -0,0 +1,25 @@
+---
+title: eureka
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+Apache APISIX 支持使用 Eureka 做服务发现。
```

Comment:
What about adding a hyperlink for Eureka here?

Reply:
okay~ done~ thanks~ @tokers

Reply:
Actually, I mean add a hyperlink about what is Eureka.

Reply:
okay~ got~ done~ thanks~ @tokers